cvmfs/sync_union_tarball.cc

```diff
@@ -100,6 +107,9 @@ void SyncUnionTarball::Traverse() {
     }
   }
 
+  /* we are simplying deleting entity from  the repo*/
```

Comment:
Nit: please use `//` style comments (in order to allow quickly commenting larger blocks with `/* */` if needed)